README.md

```diff
@@ -50,14 +50,48 @@ The `tests.conf` file contains configuration telling the CTF what extra remote t
 ### environment.py
 You can implement some of the methods that are typically used with Behave inside this file. It will be combined with the CTF common `environment.py` file. This integration may not be perfect yet, feel free to test and provide feedback.
 
-## CLI tool
-The key part of the framework is the CLI tool called `ctf-cli`. It gathers information, reads configurations, sets up the environment for Behave and runs it. Currently `ctf-cli` tool supports only ansible for running commands on some host. To run your tests make sure you included configuration for ansible in the `ctf-cli` configuration.
+## Getting Started
 
 ### Requirements
 * git
 * behave >= 1.2.5
 * ansible
 
+### Installation
+
+1. Clone repositories
+```
+git clone https://github.com/Containers-Testing-Framework/ctf-cli.git
+git clone https://github.com/Containers-Testing-Framework/common-steps.git
```

Comment:
Cloning common-steps and common-features is not required - it would be performed during cli-launch